lib/elix/brain/redis_store.ex

```diff
@@ -0,0 +1,65 @@
+defmodule Elix.Brain.RedisStore do
+  @moduledoc """
+  A Redis back-end for the Brain which persists memory to disk.
+  """
+
+  @behaviour Elix.Brain.Store
+  @encoded_binary_prefix "binterm:"
+
+  def start_link do
+    redis_url = System.get_env("REDIS_URL") || "redis://"
```

Comment:
I should perhaps also check for `Application.get_env(:brain, :redis_url)` to allow configuration via `config.exs` in case folks don't want to use ENV vars.